frontend/public/actions/features.ts

```diff
@@ -49,6 +50,16 @@ const detectOpenShift = dispatch => coFetchJSON(openshiftPath)
       : handleError(err, FLAGS.OPENSHIFT, dispatch, detectOpenShift)
   );
 
+const detectBaremetalPlatform = dispatch =>
+  k8sGet(InfrastructureModel, 'cluster')
+    .then((infra: K8sResourceKind) =>
+      getInfrastructurePlatform(infra) === 'BareMetal' && dispatch(setFlag(FLAGS.BAREMETAL, true)))
```

Comment:
Maybe I'm missing something, but shouldn't we set the flag like this?

```ts
(infra: K8sResourceKind) => dispatch(setFlag(FLAGS.BAREMETAL, getInfrastructurePlatform(infra) === 'BareMetal'))
```

Reply:
Done